lib/main.coffee

```diff
@@ -43,12 +43,12 @@ module.exports =
       name: 'coala'
       grammarScopes: ['*']
       scope: 'file'
-      lintOnFly: true
+      lintsOnChange: true
       lint: (textEditor) =>
         filePath = textEditor.getPath()
         parameters = []
         parameters.push '--find-config'
-        parameters.push '--limit-files=' + filePath
+        parameters.push '--limit-files=' + path.basename(filePath)
```

Comment:
as i said, coala will be executed with workingDirectory = directory, which contains file for analysis. So, we need to pass only file name in this direcory